xm.js

```diff
@@ -364,8 +365,8 @@ function MixChannelIntoBuf(ch, start, end, dataL, dataR) {
   var volE = ch.volE / 64.0;    // current volume envelope
   var panE = 4*(ch.panE - 32);  // current panning envelope
   var p = panE + ch.pan - 128;  // final pan
-  var volL = volE * (128 - p) * ch.vol / 8192.0;
-  var volR = volE * (128 + p) * ch.vol / 8192.0;
+  var volL = volE * (player.xm.global_volume - p) * ch.vol / 8192.0;
+  var volR = volE * (player.xm.global_volume + p) * ch.vol / 8192.0;
```

Comment:
wait, nevermind about the gainNode. that won't work, as it won't be synchronized with the actual playback.

so this just needs to be
```
  var volL = player.xm.global_volume * volE * (128 - p) * ch.vol / 1048576.0;
  var volR = player.xm.global_volume * volE * (128 + p) * ch.vol / 1048576.0;
```
(or maybe it should just be `/ (64.0 * 128 * 128)`)